src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/StartupCode/StartupCodeHelpers.Extensions.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Internal.Runtime.CompilerHelpers
 {
-    public partial class StartupCodeHelpers
+    internal partial class StartupCodeHelpers
```

Comment:
Methods on this type are referenced by ilc. Do we need to prevent IL linker from stripping them?

Reply:
For iOSlike scenarios? As far as I know Corelib is fully rooted because there's a lot more that should not be stripped, not just the publics. Cc @ivanpovazan

Reply:
For iOS-like builds we run ILLink+ILC in sequence passing `System.Private.CoreLib` untouched to both of the tools. Basically, even if ILLink trims it, we pass a fresh one to ILC.